src/Pages/ProfilePage/ProfilePage.jsx

```diff
@@ -0,0 +1,14 @@
+import { UserDetails } from './Components/UserDetails';
+import { UserPost } from './Components/UserPost';
+
+const ProfilePage = () => {
+	return (
+		<div className='app-content'>
+			<UserDetails />
+			<hr />
```

Comment:
suggestion: handle the `<hr />` part in css

Reply:
Any reason behind this?

Reply:
read #2 here:

https://dev.to/masakudamatsu/everything-about-hr-when-to-use-it-and-how-to-style-it-2nk2